base_multi_company/hooks.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
```

Comment:
@pedrobaeza - some of this code was licensed under AGPL in your `partner_multi_company`. I created this module as LGPL, so it got included here as such. Are you good with switching the parts I scavenged from AGPL or would you like me to switch this module?